### PR TITLE
(#125) notifications service implementation -- external events arrangement.

### PR DESCRIPTION
**Pull request by SaintAngeLs**
Providing the notification service implementation.

### CODE REVIEW

**Comment by eggwhat** [comment]
Wouldn't it be easier/better to create notifications by using external events and their handlers? I.e. if we want to create a notification that someone has invited you to be your friend, utilize a FriendInvited event from friends service.

**Comment by SaintAngeLs** [comment]
This PR will be updated with the full implementation of the external events subscription for other friends 'operations'.

**Comment by SaintAngeLs** [comment]
In the next PR update I will provide the blazor application update regarding to the notifications service.

File: MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Api/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:5007"
```
Comment (an2508374) [inline MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Api/Properties/launchSettings.json:6]: I think there we should have port 5006.

File: MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Api/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": false,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "local"
+      }
+    },
+    "MiniSpace.Services.Students": {
```
Comment (an2508374) [inline MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Api/Properties/launchSettings.json:17]: And there we should have Notifications.

File: MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Application/Services/Clients/IEventsServiceClinet.cs
Comment (an2508374) [inline MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Application/Services/Clients/IEventsServiceClinet.cs:1]: Probably a typo in file name.

**Comment by an2508374** [comment]
I resolved conflict merge which appeared in Startup.cs file after adding Notifications and Reaction Services there.

**Review by an2508374 — APPROVED** [review]

**Pull request merged**